### PR TITLE
Migrate objc_framework to apple_static_framework_import

### PR DESCRIPTION
**Pull request by thii**

### CODE REVIEW

File: Sources/PodToBUILD/ObjcLibrary.swift [hunk 1]
```diff
@@ -89,26 +89,29 @@ public struct ConfigSetting: BazelTarget {
...
+public struct AppleStaticFrameworkImport: BazelTarget {
     let name: String // A unique name for this rule.
     let frameworkImports: AttrSet<[String]> // The list of files under a .framework directory which are provided to Objective-C targets that depend on this target.
 
     var acknowledged: Bool {
         return true
     }
 
-    // objc_framework(
+    // FIXME: provide an API for apple_dynamic_framework_import.
```
Comment (rahul-malik) [inline Sources/PodToBUILD/ObjcLibrary.swift]: Can you file an issue for this and put a link to that issue in the comments here?

Comment (thii) [reply Sources/PodToBUILD/ObjcLibrary.swift]: I just moved these comments from

https://github.com/pinterest/PodToBUILD/blob/7524dd21cd72067ee15b0564da90aed0be79d3ae/Sources/PodToBUILD/ObjcLibrary.swift#L121-L124.

Comment (rahul-malik) [reply Sources/PodToBUILD/ObjcLibrary.swift]: Got it. Seems like the FIXME isn't really needed if we are intentionally declaring that all frameworks are static.

File: Sources/PodToBUILD/ObjcLibrary.swift [hunk 2]
```diff
@@ -89,26 +89,29 @@ public struct ConfigSetting: BazelTarget {
     }
 }
 
-// https://bazel.build/versions/master/docs/be/objective-c.html#objc_framework
-public struct ObjcFramework: BazelTarget {
+// https://github.com/bazelbuild/rules_apple/blob/818e795208ae3ca1cf1501205549d46e6bc88d73/doc/rules-general.md#apple_static_framework_import
+public struct AppleStaticFrameworkImport: BazelTarget {
```
Comment (rahul-malik) [inline Sources/PodToBUILD/ObjcLibrary.swift]: We will need to add a load statement into files that use this target type.

I've made this a lot easier to do in https://github.com/pinterest/PodToBUILD/pull/68 so lets merge that first and then update this PR

**Comment by rahul-malik** [comment]
@thii - I've merged the commit for updating the bundle resource rules.

If you look at https://github.com/pinterest/PodToBUILD/commit/ee8466e91bc5621f148ec9809cef754d5ecb81da#diff-8afbbb3f3499e3d5b95f672b3618a839R111 you will see how to add load statements for your rules. 

Essentially all you do is add another line or two to see if there is a rule of the type you've added, if so supply the load statement for it

**Pull request merged**